week-1-practise/level-1/4-logic-error-1.js

```diff
@@ -1,7 +1,7 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  return word.trim("  CodeYourFuture ");
```

Comment:
According to this [documentation](https://www.w3schools.com/jsref/jsref_trim_string.asp) the `trim()` method doesn't take any parameters. How would you fix your code then?